tools/db/build/sp_keyboard_search_by_language_bcp47_tag.sql

```diff
@@ -16,7 +16,7 @@ CREATE PROCEDURE sp_keyboard_search_by_language_bcp47_tag (
 ) AS
 BEGIN
   DECLARE @varTag NVARCHAR(250)
-  SET @varTag = dbo.f_get_canonical_bcp47(@prmTag)
+  SET @varTag = $schema.f_get_canonical_bcp47(@prmTag)
```

Comment:
`$schema` is replaced by the `sqlrun()` function with the name of the active schema. This is the only place where the schema reference is required due to a quirk in SQL Server needing reference to the schema when calling a function...